README.md

```diff
@@ -13,17 +13,25 @@ Streamlink is a fork of the [livestreamer](https://github.com/chrippa/livestream
 
 Streamlink can be installed either via: `pip` or manually via `python setup.py install`.
 
-Pip:
+**Python Pip:**
```

Comment:
Could we do `Python via pip` here so it's a complete phrase? Also no need to capitalize pip.